package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hubspot/cli",
-  "version": "6.3.0",
+  "version": "6.3.0-experimental.0",
```

Comment:
We need to change this back before merging

Reply:
Good call. Seems like this is a slightly annoying side-effect of the experimental releases, but it's not awful

Reply:
Yeah, I was thinking this could be something we just catch in PRs. But I could also update the script to revert the version locally after you publish an experimental release

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1123/commits/4d29cfa5d53a3cd02049061caea52a460d5ef11f